acquia_lift.routing.yml

```diff
@@ -4,4 +4,47 @@ acquia_lift.admin_settings_form:
     _form: '\Drupal\acquia_lift\Form\AdminSettingsForm'
     _title: 'Acquia Lift settings'
   requirements:
-    _permission: 'administer acquia lift'
+    _permission: 'administer acquia_lift'
+
+acquia_lift.slot.overview:
+  path: '/admin/config/content/acquia-lift/slots'
+  defaults:
+    _title: 'Slots'
+    _entity_list: 'acquia_lift_slot'
+  requirements:
+    _entity_create_access: 'acquia_lift_slot'
+
+entity.acquia_lift_slot.add_form:
+  path: '/admin/config/content/acquia-lift/slots/add-facet'
+  defaults:
+    _entity_form: 'acquia_lift_slot.default'
+  requirements:
+    _entity_create_access: 'acquia_lift_slot'
+
+entity.acquia_lift_slot.edit_form:
+  path: '/admin/config/content/acquia-lift/slots/{acquia_lift_slot}'
+  defaults:
+    _entity_form: 'acquia_lift_slot.edit'
+  requirements:
+    _entity_access: 'acquia_lift_slot.edit'
+
+entity.acquia_lift_slot.enable:
+  path: '/admin/config/content/acquia-lift/slots/{acquia_lift_slot}/enable'
```

Comment:
Not in favor, same reasoning as above that /admin/config/content/acquia-lift/slot does not exist. Also not in line with examples and other contrib modules